settings.php

```diff
@@ -8,11 +8,23 @@
  * Database configuration:
  *
  * Most sites can configure their database by entering the connection string
- * below. If using master/slave databases or multiple connections, see the
+ * below. If using primary/replica databases or multiple connections, see the
  * advanced database documentation at
  * https://api.backdropcms.org/database-configuration
```

Comment:
The URL changed in the meantime to `https://docs.backdropcms.org/documentation/database-configuration`.